### ISSUE TITLE
`Room.onMessage` does not return the unbind listener function

### ISSUE THREAD

**Issue by domtes** [colyseus/colyseus.js#83]
I want to manage a room message only once, i.e. removing the listener once it has been called, and I'm trying to do something like this:

```
const unbind = this.onMessage('m', (c, m) => {
    // ....
    unbind();
});
```

But I get an error (`unbind` is not a function).

Looking at [the source code](https://github.com/colyseus/colyseus.js/blob/master/src/Room.ts#L113) and at [nanoevents](https://github.com/ai/nanoevents) I would have expected the `unbind` function to be set.

**Comment by endel** [comment]
Hi @domtes, just to confirm: you're referring to the server-side usage, right? The client-side (this repository) does support what you've mentioned here, but the server-side doesn't.

**Comment by domtes** [comment]
Yes, of course I meant server side but I forgot to mention that.
